apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_egress.erl

```diff
@@ -42,32 +45,54 @@
 config(#{remote := RC = #{}} = Conf) ->
     Conf#{remote => emqx_bridge_mqtt_msg:parse(RC)}.
 
--spec send(pid(), message(), egress()) -> ok.
-send(Pid, MsgIn, Egress) ->
-    emqtt:publish(Pid, export_msg(MsgIn, Egress)).
+-spec send(pid(), trace_rendered_func(), message(), egress()) -> ok.
```

Comment:
wrong spec ?

Reply:
Looks correct to me and dialyzer did not complain. Why do you think it is incorrect?

Reply:
it now also returns {unrecoverable_error, Reason} which is by the way a bit odd because i suppose it should be {error, {unrecoverable_error, Reason}}

Reply:
Thanks. I see now. Will fix. The returned {unrecoverable_error, Reason} is converted to {error, {unrecoverable_error, Reason}} in the connector so it still worked but I will fix this as well to make it less confusing.

Reply:
Fixed.